_includes/configuration/authentication.html

```diff
@@ -201,3 +201,35 @@
   }
 }</pre>
 </div>
+
+<h3>Mixed Authentication</h3>
+
+<p>
+Mixed authentication allows collins to use multiple authentication backends
+simultaneously. An example use-case could be a scenario where real users live in
+LDAP but system users (for automation etc.) are defined using file
+authentication.
+</p>
+<p>
+To use mixed authentication, simply specify a list of authentication backends as
```

Comment:
Does the order of the list set the search order for the authentication backends or are they searched in parallel?  What happens if the same name is in multiple backends, i.e. which one wins?